docs/start/hello-world.ipynb

```diff
@@ -41,10 +62,10 @@
    "id": "21f7a26c",
    "metadata": {},
    "source": [
-    "The following code cell creates a circuit that produces a *Bell state,* which is a specific two-qubit entangled state.\n",
+    "The following code cell creates a circuit that produces a *Bell state,* which is a state wherein two (or more) qubits are fully entangled with each other.\n",
     "\n",
     "<Admonition type=\"note\" title=\"Note: bit ordering\">\n",
-    "  The Qiskit SDK uses the LSb 0 bit numbering where the $n^{th}$ digit has value $1 \\ll n$ or $2^n$. Because we usually write numbers on paper with the most significant digits to the left and the least significant digits to the right (in the Hindu-Arabic system used in most of the world), this has the consequence that the bits are labeled with indices increasing <b>from right to left</b>. This LSb 0 convention makes mathematics easier and is the most commonly used for modern digital electronics, although the opposite convention MSb 0 is also found in some domains. Converting the $i^{th}$ index between LSb 0 and MSb 0 conventions on an $n$-bit register is as simple as $i \\rightarrow n-i-1$. This differs across authors and software packages, so be aware! For more details, see the [Bit-ordering in the Qiskit SDK](../build/bit-ordering) topic.\n",
+    "  The Qiskit SDK uses the LSb 0 bit numbering where the $n^{th}$ digit has value $1 \\ll n$ or $2^n$. Because you usually write numbers with the most significant digits to the left and the least significant digits to the right (in the Hindu-Arabic system used in most of the world), this has the consequence that the bits are labeled with indices increasing <b>from right to left</b>. This LSb 0 convention makes mathematics easier and is the most commonly used for modern digital electronics, although the opposite convention MSb 0 is also found in some domains. Converting the $i^{th}$ index between LSb 0 and MSb 0 conventions on an $n$-bit register is as simple as $i \\rightarrow n-i-1$. This differs across authors and software packages, so be aware! For more details, see the [Bit-ordering in the Qiskit SDK](../build/bit-ordering) topic.\n",
```

Comment:
In the interest of streamlining - and since we have the entire Bit-ordering page - I might suggest to keep just the first and last sentences and remove the rest (since it points to the bit-ordering page)... so this line would be "The Qiskit SDK uses the LSb 0 bit numbering where the $n^{th}$ digit has value $1 \\ll n$ or $2^n$. For more details, see the [Bit-ordering in the Qiskit SDK](../build/bit-ordering) topic."

Reply:
Seems reasonable